docs/tutorials/load-test-grpc/ghaction.md

```diff
@@ -4,58 +4,27 @@ template: main.html
 
 # Iter8 GitHub Action
 
-The `load-test-grpc` experiment can be invoked as part of a GitHub Actions workflow using [the Iter8 Action](https://github.com/iter8-tools/iter8-action).
+To load test, benchmark and validate gRPC services with service-level objectives (SLOs), invoke a  `load-test-grpc` experiment in a GitHub Actions workflow using the [Iter8 Action](https://github.com/iter8-tools/iter8-action).
 
 ## Basic example
 
-The following example demonstrates the use of the Iter8 Action to benchmark and validate a gRPC service. This is the GitHub Actions equivalent of the [CLI-driven basic gRPC example](basicusage.md#basic-example).
-
 ```yaml
-# An earlier step in the workflow is assumed to have started the gRPC service
-# Configure experiment using a YAML file
-- run: |
-    cat << EOF > myvalues.yaml
-        host: 127.0.0.1:50051
-        call: helloworld.Greeter.SayHello
-        protoURL: https://raw.githubusercontent.com/grpc/grpc-go/master/examples/helloworld/helloworld/helloworld.proto
-    EOF
-# Run Iter8 experiment
 - uses: iter8-tools/iter8-action@v1
   with:
     chart: load-test-grpc
-    valuesFile: myvalues.yaml
+    valuesFile: experiment-config.yaml
 ```
 
-## Experiment configuration
-
-The [basic usage tutorial](basicusage.md) describes various configurable parameters of the `load-test-grpc` experiment. Their default values are documented in the chart's `values.yaml` file and can be viewed as follows.
-
-```shell
-iter8 hub -c load-test-grpc
-cat load-test-grpc/values.yaml
-```
-
-## Automated SLO validation
-
-You can specify [service-level objectives (SLOs)](basicusage.md#metrics-and-slos) as part of the `load-test-grpc` experiment. If the SLOs are not satisfied, then the action will fail. You can disable this behavior by setting the `validateSLOs` input to `false`.
+A sample `experiment-config.yaml` is:
 
 ```yaml
-- uses: iter8-tools/iter8-action@v1
-  with:
-    chart: load-test-grpc
-    valuesFile: myvalues.yaml
-    validateSLOs: false
+# An earlier step in the workflow is assumed to have started the gRPC service
+host: 127.0.0.1:50051
+call: helloworld.Greeter.SayHello
+protoURL: https://raw.githubusercontent.com/grpc/grpc-go/master/examples/helloworld/helloworld/helloworld.proto
 ```
 
-## Logging
-
-The Iter8 Action logs contain the following.
-
-- output of `iter8 version`
-- the `experiment.yaml` file generated by Iter8
-- logs from the experiment run
-- output of `iter8 report`
-- output of `iter8 assert -c completed -c nofailures -c slos`
+Details of the configuration parameters that can be set are [here](https://github.com/iter8-tools/hub/blob/main/charts/load-test-grpc/values.yaml).
```

Comment:
Details are in basicusage.md. It is the default values of the params that is here: https://github.com/iter8-tools/hub/blob/main/charts/load-test-grpc/values.yaml